llama_stack/providers/adapters/inference/ollama/ollama.py

```diff
@@ -23,6 +23,8 @@
 # TODO: Eventually this will move to the llama cli model list command
 # mapping of Model SKUs to ollama models
 OLLAMA_SUPPORTED_SKUS = {
+    "Llama-Guard-3-1B": "xe/llamaguard3:latest",
```

Comment:
this should be `Llama-Guard-3-8B` right?

Reply:
Yes. Will fix that.